package.json

```diff
@@ -26,18 +26,23 @@
   "dependencies": {
     "@babel/core": "^7.14.6",
     "@babel/plugin-proposal-class-properties": "^7.17.12",
+    "@cucumber/cucumber": "8.6.0",
```

Comment:
⚠️ I thought the user had to provide cucumber. See DEVX-1803. If you provide cucumber prepackaged, the user will never be able to add any additional cucumber related dependencies. Or did you find a workaround for this?